src/Parsers/ParserCreateQuery.h

```diff
@@ -190,6 +192,9 @@ bool IParserColumnDeclaration<NameParser>::parseImpl(Pos & pos, ASTPtr & node, E
         /// should be followed by an expression
         if (!expr_parser.parse(pos, default_expression, expected))
             return false;
+    } else if (s_ephemeral.ignore(pos, expected)) {
```

Comment:
[Style check](https://s3.amazonaws.com/clickhouse-test-reports/34424/465dee928cc0be24d9889fc975ce477754794124/style_check__actions_/style_output.txt) tells that we use newline before `{` and after `}`